### PR TITLE
更新

### PR DESCRIPTION
**Pull request by Ocxs**
加了一些东西，不过都加注释了，不影响源代码

### CODE REVIEW

File: ch05/ex5_14.cpp [hunk 1]
```diff
@@ -19,3 +19,51 @@ int main()
...
+		cout << " there is no  duplicated string." << endl;
+	}
+	else
+	{
+		cout << "the word " << bigStr << " occurred  " << bigNum << "times." << endl;
+	}
+	system("pause");
+	return 0;
+}
+*/
```
Comment (pezy) [inline ch05/ex5_14.cpp]: The codes have bugs, check following two case:

1. >the the woo woo huh huh
the word woo occurred  3times.

2. > tow tow tow
 there is no  duplicated string.

Comment (pezy) [inline ch05/ex5_14.cpp]: 3. input nothing and type `Ctrl+Z`(Windows) or `Control+D`(Mac OS) will crash.

Comment (Ocxs) [inline ch05/ex5_14.cpp]: Yes, you're right . I tried to fix it . And I fixed #1 #2 bug,but I can't fix #3 bug, please help me !

**Comment by pezy** [comment]
@Ocxs  Awesome! Thx for your contribute~ :100:

**Comment by pezy** [comment]
@Ocxs 

some advises about code style:

1. don't use `use namespace std;` see [this](http://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice) for more information.
2. **whitespace** formatting:
```cpp
while (cin>>str) // bad
while (cin >> str) // better
          ^  ^
while (svec.size()>0&&it != svec.end() - 1) // bad
while (svec.size() > 0 && it != svec.end()-1) // better

if (bigNum==1) // bad
if (bigNum == 1) // better
```

Looking forward to more contribution from you. :beers:

**Comment by Ocxs** [comment]
Thx for your advice ,I will ! :+1:

**Pull request merged**